lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsStorageFactory.java

```diff
@@ -89,7 +88,7 @@ public Storage create(ConnectorIdentity identity)
                 }
             }
             else {
-                credentials = jsonGoogleCredential.orElseThrow(() -> new VerifyException("GCS credentials not configured"));
+                credentials = jsonGoogleCredential.orElse(GoogleCredentials.getApplicationDefault());
```

Comment:
use `orElseGet`

Reply:
oh thanks! that's what was causing the test failure, good catch! Extracted to https://github.com/trinodb/trino/pull/20827